game/screen_players.hh

```diff
@@ -1,5 +1,5 @@
 #pragma once
-
+#include <unordered_map>
```

Comment:
would be a better idea to include after local includes

Reply:
Fixed :)